label_studio/users/views.py

```
@@ -104,61 +176,221 @@ def user_signup(request):
+        "users/new-ui/user_login.html",
+        {
+            "form": form,
+            "next": quote(next_page),
+            "client_id": auth0_id,
+            "redirect_uri": google_oauth_redir_uri,
+            "auth0_domain": auth0_domain,
+        },
+    )
+
+
+def google_callback_handler(request):
```

Pylint (3.3.7)

[refactor] Too many local variables (17/15)

(R0914)